opentelemetry-sdk/src/opentelemetry/sdk/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = "0.1.dev0"
```

Comment:
Should you move the API version file out of `internal/` to match?

Reply:
Do you mean something like `opentelemetry/api/version.py`? We decided to put `version.py` in internal since there is no public interface. I'm open to suggestions.

Reply:
I take https://github.com/open-telemetry/opentelemetry-python/pull/19#pullrequestreview-252462864 to mean that we can't have a single top-level version file, but separate files under `api/` and `sdk/` should be fine.